### PR TITLE
hi.js update as per v5.4.4

### PR DESCRIPTION
**Pull request by Inspire-24**
Did the ones which were looking good, few others looked quite awkward so left them

### CODE REVIEW

**Comment by Inspire-24** [comment]
https://github.com/Inspire-24/krunker.io-translations/blob/0bb2e2276c8dda5d4f82e0568c96ff44c9742a3a/hi.js#L978

File: hi.js [hunk 1]
```diff
@@ -922,61 +922,61 @@ chall.name.cc Get {0} close-calls {1}
...
+ranked.switch खेल के बीच मे नहीं बदल सकते
 ranked.spectate Spectating Disabled in Ranked
-ranked.login Login to Spawn
+ranked.login रेस्पॉन करने के लिए लॉगिन करें
 ranked.sync Missing Sync Data
-ranked.team.spawn Please join a Team to Spawn
-ranked.class.spawn Please select a Class to Spawn
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
+ranked.team.spawn कृपया रेस्पॉन करने के लिए एक टीम जॉइन करें
```
Comment (Codewith-Akshut) [inline hi.js:931]: Try updating रेस्पॉन to स्पॉन here and the line below this. Rest looks good!!

**Comment by Codewith-Akshut** [comment]
All looks great in my opinion.

**Comment by Inspire-24** [comment]
@mty22 ready for the merge :)

File: hi.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ settings.client.resetSettings रीसेट क्लाइंट सेटि
 settings.client.acceleratedCanvas एक्सेलरेटेड कैन्वस
 settings.client.angleBackend ऐंगल ग्राफ़िक्स बैकेंड
 settings.client.colorProfile रंग प्रोफ़ाइल
-settings.client.resourcePacks Resource Packs
+settings.client.resourcePacks Resource Packs
```
Comment (mty22) [inline hi.js:117]: ill fix this up before merge

File: hi.js [hunk 3]
```diff
@@ -922,61 +922,61 @@ chall.name.cc Get {0} close-calls {1}
...
+ranked.login स्पॉन करने के लिए लॉगिन करें
 ranked.sync Missing Sync Data
-ranked.team.spawn Please join a Team to Spawn
-ranked.class.spawn Please select a Class to Spawn
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
+ranked.team.spawn कृपया स्पॉन करने के लिए एक टीम जॉइन करें 
+ranked.class.spawn कृपया स्पॉन करने के लिए एक क्लास सेलेक्ट करें 
+modes.round.win आपकी टीम ने यह राउंड सिक्युर कर लिया है 
+modes.round.loss यह राउंड अब जीता नहीं जा सकता
```
Comment (mty22) [inline hi.js:934]: and these

**Comment by mty22** [comment]
@Inspire-24 just keep an eye out for whitespace in future :)

Check out my change here: https://github.com/mty22/krunker.io-translations/pull/357/commits/b894329c278ab6116c0dda385c680e97c8974427

File: hi.js [hunk 4]
```diff
@@ -649,12 +649,12 @@ windows.secondary.unlocked उनलॉक्ड
 windows.secondary.customOnly सिर्फ कस्टम्स
 windows.secondary.premiumOnly सिर्फ प्रीमियम
 windows.secondary.req {0} पर अनलॉक हो जाता है
-windows.skins.unlocked Unlocked
-windows.skins.customOnly Customs Only
-windows.skins.premiumOnly Premium Only
-windows.skins.classOnly Req {0}
-windows.skins.req Req lvl {0}
-windows.skins.default Default Skin
+windows.skins.unlocked उनलॉक्ड
```
Comment (HistoriFy) [inline hi.js:652]: should be अनलॉक्ड (unlocked) not उनलॉक्ड (oonlocked)

File: hi.js [hunk 5]
```diff
@@ -922,61 +922,61 @@ chall.name.cc Get {0} close-calls {1}
...
+commands.noclip.off Noclip - डिसेबल्ड
+commands.god.on गॉड मोड - इनेबल्ड
+commands.god.off गॉड मोड - डिसेबल्ड
+commands.god.on.other गॉड मोड - {0} के लिए इनेबल्ड
+commands.god.off.other गॉड मोड - {0} के लिए डिसेबल्ड
+commands.smite.on Smite - इनेबल्ड
+commands.smite.off Smite - डिसेबल्ड
+commands.smite.on.other Smite - {0} के लिए इनेबल्ड
+commands.smite.off.other Smite - {0} के लिए डिसेबल्ड
+commands.perk दिया {0} {1}
```
Comment (HistoriFy) [inline hi.js:955]: feels incomplete. should be **{1} को {0} दिया** (or something similiar

File: hi.js [hunk 6]
```diff
@@ -922,61 +922,61 @@ chall.name.cc Get {0} close-calls {1}
...
-commands.forfeit.next Must play atleast 1 round
-commands.forfeit.wait Wait till match starts to forfeit
-commands.forfeit.later Please try again in a few seconds
-commands.forfeit.voted You already voted.
-commands.forfeit.limit You can only start one vote per game
-commands.forfeit.ended Forfeiting Game
+commands.forfeit.next 1 राउंड खेलना ज़रूरी है 
+commands.forfeit.wait forfeit करने के लिए मैच शुरू होने दें 
+commands.forfeit.later कृपया कुछ देर बाद कोशिश करें 
+commands.forfeit.voted आप पहले वोटे कर चुके हैं
```
Comment (HistoriFy) [inline hi.js:972]: what is वोटे? should be **आप पहले ही वोट कर चुके हैं**

**Review by HistoriFy — COMMENTED** [review]
Reviewed few errors in the latest commit

File: hi.js (hunk 4 above)
Comment (Inspire-24) [reply hi.js:652]: typo mb

File: hi.js (hunk 6 above)
Comment (Inspire-24) [reply hi.js:972]: another typo, typing with the microsoft indic tool messes up sometimes

**Pull request merged**